From dbbcd4c625a6436477f2358dcbe7b3269c70862e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Wed, 12 Nov 2025 11:29:36 +0100
Subject: [PATCH 057/133] cmake: add a - missed in
 0f71a9291a7400844d95d7ae6aabeaa6b9191527
X-Developer-Signature: v=1; a=openpgp-sha256; l=659; i=xtex@astrafall.org;
 h=from:subject; bh=WljDcTfubOGQ+12zwaZG7P5FQUnF5igW61saNaXbeYY=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78pj2LesU2CJowp9q/fl/YE7X66vcoZVnT5b/mrb10
 Dq3iHLVjoksDGJcDJZiiixFhg3erDrp/KLLymVh5rAygQyRFmlgYGBgYGHgy03MKzXSMdIz1TbU
 MzTUMdIxYuDiFICp7u9gZFhi+nHStnrWii+pU/Q7r3Vt/3nxRHhb2IQzNkKHvMMvuvkw/M/L8Dn
 cmjMliTvkPZfNyyXLGqSVf0758L1/T/+eI0vlM1kA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit ee0a0f119c25cb5e4436aa705f672316e80a74d1)
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index daa5d4237d0f..4356fbcf0bfb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -786,9 +786,9 @@ elseif(MINGW)
   set(ZIG_HOST_TARGET_ABI "-gnu")
 elseif(ZIG_HOST_TARGET_IS_ANDROID)
   if(ZIG_HOST_TARGET_ARCH STREQUAL "arm")
-    set(ZIG_HOST_TARGET_ABI "androideabi")
+    set(ZIG_HOST_TARGET_ABI "-androideabi")
   else()
-    set(ZIG_HOST_TARGET_ABI "android")
+    set(ZIG_HOST_TARGET_ABI "-android")
   endif()
 endif()
 
-- 
2.52.0

